tests/cxx/include_with_using.cc

```diff
@@ -41,7 +41,6 @@ void TestDeclFromD4() {
   var_in_d4 = 0;
 }
 
-// IWYU: i1::UsingInCc needs a declaration
```

Comment:
This is also really surprising to me, I don't understand which of your changes made IWYU stop diagnosing on this.